test/user.js

```diff
@@ -0,0 +1,223 @@
+        User.findAll().then(u => {
+          assert.isNotEmpty(u, 'Se creo un usuario');
+        });
+        done();
+      });
+  });
+
+  it('should fail because email is in use', done => {
+    User.createModel({
+      firstName: 'firstName',
+      lastName: 'lastName',
+      username: 'username',
+      password: 'password',
+      email: 'email1@wolox.com.ar'
+    });
+    chai
+      .request(server)
+      .post('/users')
+      .send({
+        firstName: 'firstName',
+        lastName: 'lastName',
+        username: 'username',
+        password: 'password',
+        email: 'email1@wolox.com.ar'
+      })
+      .catch(err => {
+        err.should.have.status(422);
+        err.response.should.be.json;
+        err.response.body.should.have.property('message');
+        err.response.body.should.have.property('internal_code');
+        User.findAll().then(u => {
+          assert.isEmpty(u, 'No se creo un usuario');
+        });
+        done();
+      });
+  });
+
+  it('should fail because email is invalid', done => {
+    chai
+      .request(server)
+      .post('/users')
+      .send({
+        firstName: 'firstName',
+        lastName: 'lastName',
+        username: 'username',
+        password: 'password',
+        email: 'email1@molox.com.ar'
+      })
+      .catch(err => {
+        err.should.have.status(422);
+        err.response.should.be.json;
+        err.response.body.should.have.property('message');
+        err.response.body.should.have.property('internal_code');
+        User.findAll().then(u => {
+          assert.isEmpty(u, 'No se creo un usuario');
+        });
+        done();
+      });
+  });
+
+  it('should fail because password is invalid', done => {
+    chai
+      .request(server)
+      .post('/users')
+      .send({
+        firstName: 'firstName',
+        lastName: 'lastName',
+        username: 'username',
+        password: 'passw',
+        email: 'email1@wolox.com.ar'
+      })
+      .catch(err => {
+        err.should.have.status(422);
+        err.response.should.be.json;
+        err.response.body.should.have.property('message');
+        err.response.body.should.have.property('internal_code');
+        User.findAll().then(u => {
+          assert.isEmpty(u, 'No se creo un usuario');
+        });
+        done();
+      });
+  });
+
+  it('should fail because not enough parameters', done => {
+    chai
+      .request(server)
+      .post('/users')
+      .send({
+        firstName: 'firstName',
+        password: 'password',
+        email: 'email1@wolox.com.ar'
+      })
+      .catch(err => {
+        err.should.have.status(400);
+        err.response.should.be.json;
+        err.response.body.should.have.property('message');
+        err.response.body.should.have.property('internal_code');
+        User.findAll().then(u => {
+          assert.isEmpty(u, 'No se creo un usuario');
+        });
+        done();
+      });
+  });
+
+  it('Login Successful', done => {
```

Comment:
you are putting all the login tests inside the describe that talks about sign up.